test/OpenTelemetry.Tests/OpenTelemetry.Tests.csproj

```diff
@@ -1,9 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>Unit test project for OpenTelemetry</Description>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
     <TargetFrameworks>net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
-    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net462</TargetFrameworks>
```

Comment:
Should we update the workflows now or will that be a seperate PR?

https://github.com/open-telemetry/opentelemetry-dotnet/blob/503d2b0e37c44133980a7fd114a3baa7a5ed3881/.github/workflows/windows-ci.yml#L17-L19

Reply:
yea will do in follow ups.